### PR TITLE
import script for south staffordshire (closes #3033)

### LINKED ISSUE #3033: Import SST-South Staffordshire District Council
EMS: Xpress DC
Files:
- `SST/2021-02-22T10:54:49.942362/Democracy_Club__06May2021.CSV`

### PR DESCRIPTION
**Pull request by smsmith97**

### CODE REVIEW

File: polling_stations/apps/data_importers/management/commands/import_south_staffs.py [hunk 1]
```diff
@@ -2,87 +2,52 @@
...
-    elections = ["parl.2019-12-12"]
-    csv_delimiter = "\t"
-    allow_station_point_from_postcode = False
-
-    def station_record_to_dict(self, record):
-
-        if record.polling_place_id == "3464":
-            record = record._replace(
-                polling_place_easting="391056", polling_place_northing="319891"
-            )
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_south_staffs.py:21]: Can you carry this fix over please. PS name is: `Village Hall, Hyde Lea` and new id is `3849`

File: polling_stations/apps/data_importers/management/commands/import_south_staffs.py [hunk 2]
```diff
@@ -2,87 +2,52 @@
...
-
-        if record.polling_place_id == "3427":
-            record = record._replace(
-                polling_place_easting="389364", polling_place_northing="295314"
-            )
-
-        return super().station_record_to_dict(record)
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_south_staffs.py:28]: Ditto: `St Bartholomew`s Church Hall` and `4044`

**Review by GeoWill — CHANGES_REQUESTED** [review]
Looks good, just a couple of changes to carry over.

**Comment by smsmith97** [comment]
Should be correct now.

**Review by GeoWill — APPROVED** [review]

**Pull request merged**